### DISCUSSION TITLE
Failed to install Tailwind LSP

### DISCUSSION THREAD

**Discussion by signheim** [helix-editor/helix#4446] [Q&A]
Hi, I am very new to the idea to use a terminal-editor and enjoy Helix so far. It gives me more feedback compared to others I tried. For I am not really into LSP-Servers I installed this: https://www.npmjs.com/package/@tailwindcss/language-server

Then I created a languages.toml like this:

[[language]]
name = "tailwindcss"
scope = "text.html"
injection-regex = "tailwind"
file-types = ["html"]
roots = ["tailwind.config.js"]
language-server = { command = "/opt/homebrew/lib/node_modules/@tailwindcss/language-server/bin/tailwindcss-language-server", args = ["--stdio"] }

And ... you know ... I do not really know what I am doing here. What I know is that it does not work at all and also the html-highlighting is gone when this toml is active.

Is there anybody who succesfully installed tailwind-lsp in Helix?

**Comment by dead10ck** [comment]
Try running the command as you have it in your config, but from your shell to see any output it gives. This helps me debug issues whenever I get up a new LS.

For the highlighting, you can put this in the language config

```toml
grammar = "html"
```

**Comment by signheim** [comment]
Thank you for this tip. In the shell there was nothing to read when I started the server. But once I try to write some tailwind-classes in a html Helix says this:

Async job failes: protocol error: MethodNotFound: Unhandled method textDocument/completion

HTML-LSP works just fine when I remove the languages.toml (as I postet it's content PLUS the grammar-line that you mentioned) it working fine with html. Once the toml is active even the html-highlighting stopped working.
